packages/expo-file-system/src/FileSystem.ts

```diff
@@ -238,7 +241,7 @@ export abstract class NetworkTask<T extends DownloadProgressData | UploadProgres
     return await ExponentFileSystem.networkTaskCancelAsync(this.uuid);
   }
 
-  protected checkIfTaskWasCanceled(): boolean {
+  protected isTaskCancelled(): boolean {
```

Comment:
I'm wondering about this change, as `isTaskCancelled` suggest it's a property of this class 🤔 
Maybe we should make it an actual property with `getter` and `setter`? 🤔

Reply:
I'm sorry that I'm changing my mind now 😅 
I've just realised it was me that suggested this change 😂